modules/mod_finder/src/Helper/FinderHelper.php

```diff
@@ -79,6 +84,52 @@ public static function getQuery($params)
         $options['filters'] = ArrayHelper::toInteger($options['filters']);
 
         // Instantiate a query object.
-        return new Query($options, Factory::getContainer()->get(DatabaseInterface::class));
+        return new Query($options, $this->getDatabase());
+    }
+
+    /**
+     * Method to get hidden input fields for a get form so that control variables
+     * are not lost upon form submission.
+     *
+     * @param   string   $route      The route to the page. [optional]
+     * @param   integer  $paramItem  The menu item ID. (@since 3.1) [optional]
+     *
+     * @return  string  A string of hidden input form fields
+     *
+     * @since   2.5
+     *
+     * @deprecated __DEPLOY_VERSION__ will be removed in 7.0
+     *             Use the non-static method getFields
+     *             Example: Factory::getApplication()->bootModule('mod_finder', 'site')
+     *                          ->getHelper('FinderHelper')
+     *                          ->getHiddenFields($route, $paramItem)
+     */
+    public static function getGetFields($route = null, $paramItem = 0)
+    {
+        return (new self())->getHiddenFields($route);
+    }
+
+    /**
+     * Get Smart Search query object.
+     *
+     * @param   \Joomla\Registry\Registry  $params  Module parameters.
+     *
+     * @return  Query object
+     *
+     * @since   2.5
+     *
+     * @deprecated __DEPLOY_VERSION__ will be removed in 7.0
+     *             Use the non-static method getSearchQuery
+     *             Example: Factory::getApplication()->bootModule('mod_finder', 'site')
+     *                          ->getHelper('FinderHelper')
+     *                          ->getSearchQuery($params, Factory::getApplication())
+     */
+    public static function getQuery($params)
+    {
+        $app = Factory::getApplication();
+
+        return $app->bootModule('mod_finder', 'site')
+                                  ->getHelper('FinderHelper')
+                                  ->getSearchQuery($params, $app);
```

Comment:
```suggestion
        return $app->bootModule('mod_finder', 'site')
                   ->getHelper('FinderHelper')
                   ->getSearchQuery($params, $app);
```
CS: Fix indentation.